Assignment 5.Rmd

```diff
@@ -75,3 +78,10 @@ D2$prediction <- predict(score_ctree, D2)
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+
+```{r}
+#to find the number of monitor 
+table(D2$prediction)
+
+```
+#The predictive model seems to be pretty good. Of these students who scored a 1 on the test, 116 were predicted to be in the "no action" group, 84 were in the "monitor" group, and none were in the "intervene" group. Since the model accurately predicted these high scoring students to be mostly in the "no action" group I would say it is a successful model. Having some students from the "monitor" group is ok.
```

Comment:
Take a look at the results of the students in the new data set - they scored 1. This suggests that any prediction that is in the monitor or intervene group is inaccurate. Do you still believe the model is good?